Composer/packages/tools/language-servers/language-understanding/src/luParser.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { fork, ChildProcess } from 'child_process';
+import path from 'path';
+
+import { parser } from '@microsoft/bf-lu/lib/parser';
+import { updateIntent, checkSection } from '@bfc/indexers/lib/utils/luUtil';
+import { luIndexer } from '@bfc/indexers';
+import { LuFile, ILUFeaturesConfig, LuIntentSection, Diagnostic } from '@bfc/shared';
+import uniqueId from 'lodash/uniqueId';
+
+export const isTest = process.env?.NODE_ENV === 'test';
```

Comment:
Does this environment test exist?

Reply:
Yes, it's still exist environment test, but we can do some mocks to update the unit tests. I can do it in a clean code PR.